tests/unit/core/statistics/datatypes/test_floattracker.py

```diff
@@ -13,6 +15,28 @@ def test_values_are_min_max():
     assert first.sum == sum(vals1)
 
 
+def test_np_float():
+    first = FloatTracker()
+    float32_array = np.array([1.0, 2.0, 3.0], dtype=np.float32)
+    for val in float32_array:
+        first.update(val)
+
+    assert first.count == len(float32_array)
```

Comment:
minor: consider creating a small test helper function: check_float_tracker(float_tracker, data) to wrap these max/min/sum checks to reduce duplication.

Reply:
looks like you did that in last revision!